src/transformState.js

```diff
@@ -6,6 +6,28 @@
  */
 function transformState(state, actions) {
   // write code here
+  const newObj = state;
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(newObj, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const i of action.keysToRemove) {
+          delete newObj[i];
+        }
+        break;
+
+      default:
+        for (const k in newObj) {
+          delete newObj[k];
+        }
```

Comment:
it is not good practice to use default case for condition
use it in `case 'removeProperties'`
and in `default` just `break`